app/bundles/CoreBundle/Assets/js/1.core.js

```diff
@@ -82,8 +83,33 @@ mQuery( document ).ready(function() {
                 });
         }
     }, mauticSessionLifetime * 1000 / 2);
+
+    // Copy code block on click
+    mQuery(document).on('click', 'code', function() {
+        var $codeBlock = mQuery(this);
+        var $icon = $codeBlock.find('.copy-icon');
+        var $temp = mQuery('<textarea>');
+        mQuery('body').append($temp);
+        $temp.val($codeBlock.text()).select();
+        document.execCommand('copy');
+        $temp.remove();
+        $icon.removeClass('ri-clipboard-fill').addClass('ri-check-line');
+        setTimeout(function() {
+            $icon.removeClass('ri-check-line').addClass('ri-clipboard-fill');
+        }, 2000);
+    });
+    initializeCodeBlocks();
 });
 
+function initializeCodeBlocks() {
+    mQuery('code').each(function() {
+        var $codeBlock = mQuery(this);
+        if (!$codeBlock.find('.copy-icon').length) {
+            $codeBlock.append('<i class="ri-clipboard-fill ml-xs copy-icon"></i>');
+        }
+    });
+}
+
```

Comment:
I'm not sure what the changes in `1.core.js` have to do with the scope of the PR (changing buttons styles). If this is new functionality, it would fit better into another PR.